ui/lib/pki/addon/templates/overview.hbs

```diff
@@ -1,23 +1,29 @@
 <SecretListHeader
-  @model={{this.model}}
+  @model={{this.model.engine}}
   @backendCrumb={{hash
-    label=this.model.id
-    text=this.model.id
+    label=this.model.engine.id
+    text=this.model.engine.id
     path="vault.cluster.secrets.backend.list-root"
-    model=this.model.id
+    model=this.model.engine.id
   }}
   @isEngine={{true}}
 />
 <Toolbar>
   <ToolbarActions>
-    <ToolbarLink @route="configuration.create">
-      Configure PKI
-    </ToolbarLink>
+    {{#unless this.model.hasConfig}}
+      <ToolbarLink @route="configuration.create">
+        Configure PKI
+      </ToolbarLink>
+    {{/unless}}
   </ToolbarActions>
 </Toolbar>
 
-<EmptyState @title="PKI not configured" @message="This PKI mount hasn’t yet been configured with a certificate issuer.">
-  <LinkTo @route="configuration.create" @model={{this.model}}>
-    Configure PKI
-  </LinkTo>
-</EmptyState>
\ No newline at end of file
+{{#if this.model.hasConfig}}
+  TODO show overview items
```

Comment:
Not sure if this supposed to be a comment - I noticed you have a TODO in `ui/lib/pki/addon/components/pki-configure-form.hbs` and it was commented out in the template :)